metaStoreApp/app/controllers/api/v1/audios_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class AudiosController < MediasController
+      def create
+        super
+        if (@@media.save)
+          render json: { status: 'SUCCESS', message: 'saved audio', data: @@media }, status: :ok
+        else
+          render json: { status: 'ERROR', message: 'failed to save audio', data: @@media.errors }, status: :unprocessable_entity
+        end
+      end
+    
+      def destroy
+        super
+        render json: { status: 'SUCCESS', message: 'Deleted audio', data: @@del_media }, status: :ok 
+      end
+
+      def duration_tc(duration)
```

Comment:
this method should be on corresponding model.
duration_tc is a property on the media object, if we are defining this in a controller we cannot use this method in any other place.